tests/NCronJob.Tests/TestHelper.cs

```diff
@@ -53,7 +54,10 @@ protected virtual void Dispose(bool disposing)
         serviceProvider?.Dispose();
     }
 
-    protected ServiceProvider CreateServiceProvider() => serviceProvider ??= ServiceCollection.BuildServiceProvider();
+    // TODO: Replace calls to this method in the tests with `ServiceProvider`
```

Comment:
Will be done slowly, test suite per test suite.
Otherwise the generated diff noise if overwhelming.

Reply:
Fair. And the name was missleading at best. It should have been at least `GetOrCreateXXX`